src/jobs/import.js

```diff
@@ -576,14 +679,29 @@ async function importJurica() {
                 await JudilibreIndex.updateDecisionDocument(normalized, null, 'skip import (already inserted)');
                 console.warn(`Jurica import issue: { sourceId: ${row._id}, sourceName: 'jurica' } already inserted...`);
```

Comment:
to remove or to change by:

```
CustomLog.log({ operationName: "ImportJuricaUpdateDecision", msg: `Jurica import issue: { sourceId: ${row._id}, sourceName: 'jurica' } already inserted...` })
```

Reply:
J'ai fait des changement en rapport avec ce commentaire